qutebrowser.el

```diff
@@ -1132,7 +1132,10 @@ CONN is the `jsonrpc-connection' the request was received on.
 METHOD is the method that was called.
 PARAMS are the parameters given."
   (cl-case method
-    (eval (eval (read (plist-get params :code))))
+    (eval
+     (with-current-buffer
+         (exwm--id->buffer (plist-get params :x11-win-id))
```

Comment:
Could you wrap the buffer in `(or (current-buffer))` so we don't error out if `:x11-win-id` is missing or it somehow can not be resolved to a buffer?

Reply:
yeah, I also added a check for `exwm-wm-mode`, so that the code works for non-exwm setups